azure-pipelines.yml

```diff
@@ -0,0 +1,49 @@
+jobs:
+- job:
+
+  strategy:
+    matrix:
+      Mojave-Xcode-10.2.1:
+        IMAGE_POOL: 'macOS-10.14'
+        XCODE_VERSION: '10.2.1'
+      Mojave-Xcode-11.0:
+        IMAGE_POOL: 'macOS-10.14'
+        XCODE_VERSION: '11'
+
+  pool:
+    vmImage: $(IMAGE_POOL)
+
+  steps:
+
+  - script: |
+      sudo xcode-select --switch "/Applications/Xcode_$(XCODE_VERSION).app/Contents/Developer"
+    displayName: "Select Xcode $(XCODE_VERSION)"
+
+  - script: xcodebuild -version
+    displayName: "Version of xcodebuild"
+  
+  - script: git clone -b develop https://github.com/calabash/FBSimulatorControl.git ../FBSimulatorControl
+    displayName: Download FBSimulatorControl dependency
+
+  - script: |
+      git clone -b master https://$(GitHubAccessToken):x-oauth-basic@github.com/xamarinhq/calabash-codesign.git ../calabash-codesign
+      ../calabash-codesign/apple/create-keychain.sh
+    displayName: "Download and install keychain"
+
+  - script: bundle install
+    displayName: "Bundle install"
+
+  - script: make frameworks
+    displayName: "Make frameworks"
```

Comment:
Remove this step.  See the note about FBSimulatorControl above.

Reply:
Done